opennms-webapp/src/main/java/org/opennms/web/services/FilterFavoriteService.java

```diff
@@ -193,11 +193,8 @@ protected boolean deleteFavorite(OnmsFilterFavorite favorite) {
         return false;
     }
 
+    @SneakyThrows(UnsupportedEncodingException.class)
```

Comment:
We replace here the kind of useless try / catch (since every Java platform has to support UTF-8) by this annotation. See more details here: https://projectlombok.org/features/SneakyThrows